src/d3a/d3a_core/live_events.py

```diff
@@ -62,6 +65,10 @@ def apply(self, area):
             return False
 
         area.children = [c for c in area.children if c.uuid != self.area_uuid]
+        if len(area.children) == 0:
+            area._markets = AreaMarkets(area.log)
+            area.stats = AreaStats(area._markets)
```

Comment:
As far as I see `stats` contain some critical infos historical market stats that are being loose here via initialising it

Reply:
@faizan2590 You are right, i removed the re-initiation of AreaStats here:
https://github.com/gridsingularity/d3a/pull/812/commits/70b9b1ffe53a035af250400c7a51ffb15b9272a7